client/app/routes/application.js

```diff
@@ -25,5 +26,10 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
     } else if (this.get('currentDistrict.isInvalid')) {
       this.replaceWith('index');
     }
+  },
+  actions: {
+    trackEvent(eventName) {
+      this.get('intercom').trackEvent(eventName);
+    }
```

Comment:
Currently elsewhere in the code we're performing Intercom actions (like boot and shutdown) by injecting the `intercom` service into each module that wants to access it. Maybe we should keep the approach consistent: either all per-module injection, or all route-actions.